lib/cli/objectlistcommand.cpp

```diff
@@ -104,6 +118,16 @@ int ObjectListCommand::Run(const boost::program_options::variables_map& vm, cons
 	Log(LogNotice, "cli")
 		<< "Parsed " << objects_count << " objects.";
 
+	auto objectsPathCtime (GetCtime(Configuration::ObjectsPath));
+	auto varsPathCtime (GetCtime(Configuration::VarsPath));
+
+	if (objectsPathCtime < varsPathCtime) {
+		Log(LogWarning, "cli")
+			<< "These data is " << Utility::FormatDuration(varsPathCtime - objectsPathCtime)
```

Comment:
Technically, "data" is a plural so "these" is correct, but then it should be consistently treated as a plural and use "are" as well. Or stick with "This data is", which feels more common to me, but the number of Google results says something different (with the first results being about which of these two options is correct).